src/components/Form/Form.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import './Form.css';
+
+export class Form extends React.Component {
+  state = {
+    title: '',
+    arrLength: 1,
```

Comment:
are you sure you need to create extra value in state?
and the name doesn't represent what value it stores